Sources/XCHammer/XcodeTarget.swift

```diff
@@ -294,20 +300,15 @@ public class XcodeTarget: Hashable, Equatable {
         case .sourceFile:
             return resolveExternalPath(for: fileInfo.subPath)
         case .generatedFile:
-            // Tulsi sticks some generated source file types in _tulsi-includes
-            // and the aspect is aware of this.
-            if useTulsiPath && fileInfo.fullPath.hasPrefix("bazel-tulsi-includes") {
-                return "tulsi-workspace/" + fileInfo.fullPath
-            } else {
-                return "bazel-genfiles/" + resolveExternalPath(for: fileInfo.subPath)
-            }
+            return "xchammer-workspace/xchammer-includes/x/x/" + resolveExternalPath(for: fileInfo.subPath)
         }
     }
 
     func resolveExternalPath(for path: String) -> String {
-        if path.hasPrefix("external/")
-            || self.label.packageName?.hasPrefix("@") ?? false {
+        if path.hasPrefix("external/") {
             return path.replacingOccurrences(of: "../", with: "external/")
```

Comment:
This line is confusing and probably error prone we may be able to clean it up / remove it